src/endpoints/create.rs

```diff
@@ -254,17 +248,27 @@ pub async fn create(
                     &file.name()
                 );
 
-                let mut f = web::block(|| std::fs::File::create(filepath)).await??;
+                let mut f = tokio::fs::File::create(filepath).await?;
```

Comment:
thank you! that made no sense before :)

Reply:
Yeah was also quite confused why you block with this the request. With tokio it is much more clear what is done there :)